applications/ParticleMechanicsApplication/custom_utilities/mpm_stress_principal_invariants_utility.h

```diff
@@ -41,7 +41,16 @@ namespace Kratos
 
             typedef unsigned int SizeType;
 
-            // Sort Principal Stresses, Strains, and Directions to magnitude order
+            // Tolerance
```

Comment:
Most of this utilities are duplicated respect constitutive_law_utilities on StructuralMechanicsApplication

Reply:
Agree with you, but I am having a bit more specialized for soil with first and second derivatives.